### ISSUE TITLE
Add a check to test if a website uses double tap for mobile devices or not

### ISSUE THREAD

**Issue by ayumihamsaki** [GoogleChrome/lighthouse#9474]
It's really annoying when a website needs to have a double tap on links and buttons when the website is being used on a mobile device.

It would be a nice feature for Lighthouse to test if the website has this poor user experience.

Then if the website does make users have to double tap for links and buttons, give a warning message in lighthouse so the developer knows there is this problem in their website.

Maybe have this feature under the Performance section.

**Comment by patrickhulce** [comment]
Thanks for filing @ayumihamsaki! Are you referring to the use of a `:hover` style [that forces double tap on iOS](https://css-tricks.com/annoying-mobile-double-tap-link-issue/)?

I'm not sure if we could go sofar as to flag all `:hover` styling served to a mobile device, but it's an interesting idea!

**Comment by ayumihamsaki** [comment]
Yes I'm talking about that issue, especially on links and buttons! For example, where I touch a link and it gets a hover underline style, but doesn't open the link and you have to touch the link a second time and then it opens the link.

I'm sure you could add some clever logic to work out the problem ones and the ok ones. It would be a useful tool to have in Lighthouse, where it scans just the webpage and flags up the problem ones as a yellow warning (not a major issue) - more like a suggestion.

**Comment by patrickhulce** [comment]
Perhaps a candidate for the UX audits @egsweeny wherever that list is being tracked?

**Comment by patrickhulce** [comment]
We don't know if this is still a problem. Someone will need to investigate.

**Comment by patrickhulce** [comment]
We like this idea if it's still a problem! Good first issue to investigate if anyone out there would like to put in the research :)

**Comment by Malusnow** [comment]
Hello, I'm interested in this issue, may I work on this?

**Comment by snehal242005** [comment]
Hi, I'd like to work on this issue!

**Comment by plutoTaco** [comment]
Is the issue still open? If it is may I work on it?

**Comment by aditya4oo4** [comment]
can you describe more about this issue any video for this issue?
